test/image/mocks/shapes.json

```diff
@@ -18,7 +18,7 @@
         "yaxis2":{"title":"category","range":[0,1],"domain":[0.6,1],"anchor":"x2","type":"category","showgrid":false,"zeroline":false,"showticklabels":false},
         "height":400,
         "width":800,
-        "margin": {"l":20,"r":20,"top":10,"bottom":10,"pad":0},
```

Comment:
I didn't want to change the image so just deleted these bad attrs uncovered when testing this fix.

Perhaps we should make a test that validates all our mocks, to ensure we're not accidentally encouraging mistaken usage like this? Not in this PR of course...

Reply:
> Perhaps we should make a test that validates all our mocks, to ensure we're not accidentally encouraging mistaken usage like this?

That's a great idea :+1: